test/metabase/lib/js_test.cljs

```diff
@@ -74,3 +74,7 @@
               {:lib/type :option/join.strategy, :strategy :right-join}
               {:lib/type :option/join.strategy, :strategy :inner-join}]
              (vec strategies))))))
+
+(deftest ^:parallel find-column-index-from-legacy-ref-test
+
+ )
```

Comment:
????

Reply:
Leftovers from development that got accidentally committed to the working branch as I was prepping to leave for vacation. There should be some tests written for the custom logic in that CLJS function since it's not a trivial wrapper of another MLv2 function.